src/components/MarkdownContent.vue

```diff
@@ -12,17 +12,9 @@
   </el-row>
 </template>
 
-<script lang="ts">
```

Comment:
I don't know why, but it caused an incomprehensible error in the schema form here. I tried for a while but couldn't solve it, so I just deleted it...

Reply:
Type '{ content: string; }' is not assignable to type 'IntrinsicAttributes & Partial<{}> & Pick<Readonly<{} & {} & {}> & VNodeProps & AllowedComponentProps & ComponentCustomProps, "style" | ... 10 more ... | "class">'.
  Property 'content' does not exist on type 'IntrinsicAttributes & Partial<{}> & Pick<Readonly<{} & {} & {}> & VNodeProps & AllowedComponentProps & ComponentCustomProps, "style" | ... 10 more ... | "class">'.

Reply:
Where is the code?